src/components/Navbar/NavbarMobile/NavbarMobile.js

```diff
@@ -0,0 +1,38 @@
+import styles from './NavbarMobile.module.css';
+import { NavLink } from 'react-router-dom';
+// import SearchIcon from '@material-ui/icons/Search';
+import LocalMallOutlinedIcon from '@material-ui/icons/LocalMallOutlined';
+import MenuOutlinedIcon from '@material-ui/icons/MenuOutlined';
+import FavoriteBorderOutlinedIcon from '@material-ui/icons/FavoriteBorderOutlined';
+
+export function NavbarMobile() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.navbarContainer}>
+        <div className={styles.logo}>
+          <NavLink to="/" className={styles.navlink}>
+            <h1>ShoeMeUp</h1>
+          </NavLink>
+        </div>
+        <div className={styles.actions}>
+          {/* <span>
+          <SearchIcon />
+        </span> */}
+          <NavLink to="/wishlist" className={styles.navlink}>
+            <span>
+              <FavoriteBorderOutlinedIcon style={{ fontSize: 27 }} />
+            </span>
+          </NavLink>
+          <NavLink to="/cart" className={styles.navlink}>
+            <span>
+              <LocalMallOutlinedIcon style={{ fontSize: 27 }} />
+            </span>
+          </NavLink>
+          <span>
+            <MenuOutlinedIcon style={{ fontSize: 30 }} />
+          </span>
```

Comment:
Give some spaces between the icon and add badges on top of the icon to find how many products are in the bag or in the wishlist.

Reply:
Thanks for pointing this out. I completely missed this part. I'll add appropriate spacing and also add the bag and wishlist counter.